music_assistant/providers/builtin_player/__init__.py

```diff
@@ -437,6 +450,15 @@ async def _serve_audio_stream(self, request: web.Request) -> web.StreamResponse:
             channels=DEFAULT_PCM_FORMAT.channels,
         )
 
+        if (user_agent := request.headers.get("User-Agent")) and "Safari" in user_agent:
+            # This is possibly an iPhone or iPad. Since Apple ignores "Accept-Ranges=none" on iOS
+            # and iPadOS for some reason, we need to slowly feed the music to avoid Safari
+            # stopping and later restarting the audio stream (from a wrong position!).
+            extra_input_args = ["-readrate", "1.0", "-readrate_initial_burst", "15"]
```

Comment:
I think it wouldn't even hurt to apply this to all requests because no doubt more devices are hurt by this.